README.md

```diff
@@ -21,7 +21,8 @@ campaign assessors to simplify the process of managing GoPhish campaigns.
 * `gophish-complete` - Completes a campaign in GoPhish and/or outputs a
   GoPhish campaign summary.
 * `gophish-export` - Exports all the data from an assessment within GoPhish
-  into a single JSON file.
+  into a single JSON file. In addition, user report JSONs for each campaign in
+  an assessment will also be generated, for later import into the PCA database.
```

Comment:
Remove reference to PCA- this repo should stand on it's own as a Gophish helper utility, regardless of whether or not it is used for PCAs.
```suggestion
  an assessment will also be exported.
```

Reply:
See 2d96175613954e3c341aa38614d80369113217af

Reply:
Thanks!  FYI, you can use [GitHub's "Commit suggestion" feature](https://docs.github.com/en/github/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/incorporating-feedback-in-your-pull-request#applying-suggested-changes) to save time when making changes like this in the future.